MoreLinq.Test/ReturnTest.cs

```diff
@@ -0,0 +1,168 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2019 Mitch Bodmer. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+using NUnit.Framework.Interfaces;
```

Comment:
Remove unused import:

```suggestion
using NUnit.Framework.Interfaces;
```

Reply:
ITestCaseData requires this now.

Reply:
Might have been the case before 236daefcdfaf33b84e7011558d422b936edebc57.